src/transport/SecureSession.h

```diff
@@ -130,6 +130,8 @@ class DLL_EXPORT SecureSession
     uint8_t mKey[kAES_CCM128_Key_Length];
 
     static CHIP_ERROR GetIV(const MessageHeader & header, uint8_t * iv, size_t len);
+
+    static CHIP_ERROR GetAAD(const MessageHeader & header, uint8_t * aad, size_t & len);
```

Comment:
could we add a comment about what this is?

Above I kind of know IV is 'initialization vector' so this seems common enough at least for me. I have no clue what an AAD is and would like to know.

Reply:
The PR says 'additional authenticated data'. Thinking that either comment, or better make the names a bit more verbose and then people reading in the future can understand.

Also 'additional data' does not tell me much. Hoping the comment could explain what this is.

Reply:
@andy31415 , updated the function name and added a comment.